lib/opal/rewriters/returnable_logic.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'opal/rewriters/base'
+
+module Opal
+  module Rewriters
+    class ReturnableLogic < Base
+      def next_tmp
+        @counter ||= 0
+        @counter += 1
+        "$ret_or_#{@counter}"
+      end
+
+      def reset_tmp_counter!
```

Comment:
Does anyone use this method? It was a part of another rewriter to simplify testing (i.e. to have consistent `tmp_name_1` in tests)

Reply:
Yeah, that's why I added it too initially, then I changed the tests in a way that doesn't require it, some cleanup is due there, maybe move the helper up the inheritance chain, I'll open an issue as a reminder 👍